### PR TITLE
[staging-17.0] Merge Master up to e87eed7a9

### PR DESCRIPTION
**Pull request by darcywong00**
5 of x merging master to staging-17.0 up to e87eed7a9

Notable changes:

* f574c6f86abbee7541b594a42d6f36dff3674be8 Replace aava1.ttf with NotoSansTamil-Regular (aava1.ttf removed in #2479)

* ekwtamil99uni.kps example has additional <kbd>[</kbd> keys from https://github.com/keymanapp/keyboards/blob/8b7824d21375a64fe32b46afa349651479d83153/release/e/ekwtamil99uni/ekwtamil99uni.keyboard_info#L12
(This is now reverted in a later commit)

### CODE REVIEW

File: experimental/gff/gff_sbs/gff_sbs.keyboard_info
Comment (mcdurdin) [inline experimental/gff/gff_sbs/gff_sbs.keyboard_info:1]: This file should not be in the PR, details should be in the .kps

File: experimental/gff/gff_sbs/source/gff_sbs.kps [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+<Package>
+  <System>
+    <KeymanDeveloperVersion>16.0.143.0</KeymanDeveloperVersion>
+    <FileVersion>7.0</FileVersion>
+  </System>
+  <Options>
+    <ExecuteProgram></ExecuteProgram>
+    <ReadMeFile>..\..\..\..\release\shared\gff\readme.htm</ReadMeFile>
+    <GraphicFile>sideimage-sheikh-bakrai-sapalo.png</GraphicFile>
+    <MSIFileName>C:\Users\Daniel Yacob\Downloads\KM-Installer-16.0.143\keymandesktop.msi</MSIFileName>
```
Comment (mcdurdin) [inline experimental/gff/gff_sbs/source/gff_sbs.kps:11]: This is probably a separate issue but the MSIFileName should not be included.

File: experimental/gff/gff_sbs/source/gff_sbs.kps [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      <Item>
+        <Name>OFL (Font Licence)</Name>
+        <FileName>OFL.txt</FileName>
+        <Arguments></Arguments>
+        <Icon></Icon>
+        <Location>psmelStartMenu</Location>
+      </Item>
+    </Items>
+  </StartMenu>
+  <Info>
```
Comment (mcdurdin) [inline experimental/gff/gff_sbs/source/gff_sbs.kps:50]: Description is missing, needs to be copied from .keyboard_info:

```suggestion
  <Info>
    <Description>This is an Oromo language input method designed for the Sheek Bakrii Saphaloo script that follows Qubee style typing rules.  It requires the included “Athinkra: Sheek Bakrii Saphaloo” font which in turn encodes the script in the Unicode Private Use Area.</Description>
```

File: release/e/ekwtamil99uni/source/ekwtamil99uni.kps [hunk 3]
```diff
@@ -201,18 +255,20 @@
...
+      <Version>2.0.5</Version>
+      <OSKFont>..\..\..\shared\fonts\noto\Taml\NotoSansTamil-Regular.ttf</OSKFont>
+      <DisplayFont>..\..\..\shared\fonts\noto\Taml\NotoSansTamil-Regular.ttf</DisplayFont>
+      <WebDisplayFonts><Font Filename="NotoSansTamil-Regular.ttf"/></WebDisplayFonts>
       <Languages>
         <Language ID="ta">Tamil</Language>
       </Languages>
       <Examples>
-        <Example ID="ta" Keys="l k s / f space a m s [ f [ d v o s" Text="தமிழ் அரிச்சுவடி" Note="&quot;Tamil alphabet&quot;" />
+        <Example ID="ta" Keys="l k s / f space a m s [ [ f [ [ d v o s" Text="தமிழ் அரிச்சுவடி" Note="&quot;Tamil alphabet&quot;" />
```
Comment (mcdurdin) [inline release/e/ekwtamil99uni/source/ekwtamil99uni.kps:266]:
```suggestion
        <Example ID="ta" Keys="l k s / f space a m s [ f [ d v o s" Text="தமிழ் அரிச்சுவடி" Note="&quot;Tamil alphabet&quot;" />

```

The `[[` in the .keyboard_info file is is an (undocumented!) escape for `[`, which is not used in the `Example` property.

**Review by mcdurdin — APPROVED** [review]
Minor tweaks required but otherwise lgtm

File: experimental/gff/gff_sbs/source/gff_sbs.kps (hunk 1 above)
Comment (darcywong00) [reply experimental/gff/gff_sbs/source/gff_sbs.kps]: Removed in 617896fbd0d64e90a8d683de295dd55770405e7c

**Pull request merged**